src/mlstacks/cli/cli.py

```diff
@@ -18,6 +18,7 @@
 from typing import Optional
 
 import click
+import pkg_resources
```

Comment:
Does this require changes in the requirements of the package or does it already use setuptools elsewhere?

Reply:
I don't think so. I did a fresh install of `mlstacks` in a new environment and it already had `setuptools` installed.